examples/governance/tests/benchmarks/BenchmarkTestBase.sol

```diff
@@ -0,0 +1,169 @@
+    bytes32 public constant IMAGE_ID = ImageID.FINALIZE_VOTES_ID;
+    bytes4 public constant MOCK_SELECTOR = bytes4(uint32(1337));
+
+    function setUp() public virtual {
+        voteToken = new VoteToken();
+        mockVerifier = new RiscZeroMockVerifier(MOCK_SELECTOR);
+        baselineGovernor = new BaselineGovernor(voteToken);
+        riscZeroGovernor = new RiscZeroGovernor(
+            voteToken,
+            IMAGE_ID,
+            mockVerifier
+        );
+    }
+
+    function generateAccounts(uint256 noOfAccounts) public noGasMetering {
+        for (uint256 i = 0; i < noOfAccounts; i++) {
+            string memory base = string(
+                abi.encodePacked("yolo", Strings.toString(i))
+            );
+            (address tempAddress, uint256 tempPk) = makeAddrAndKey(base);
+            accounts.push(tempAddress);
+            keys[tempAddress] = tempPk;
+        }
+    }
+
+    function getSignature(
+        uint8 support,
+        uint256 proposalId,
+        address signer,
+        uint256 privateKey
+    ) internal noGasMetering returns (bytes memory signature) {
+        bytes32 digest = baselineGovernor.voteHash(proposalId, support, signer);
+
+        (uint8 v, bytes32 r, bytes32 s) = vm.sign(privateKey, digest);
+        signature = abi.encodePacked(r, s, v);
+    }
+
+    function generateSignatures(
+        uint256 proposalId,
+        uint256 noOfSignatures,
+        bool isBaseline
+    ) internal noGasMetering {
+        for (uint256 i = 0; i < noOfSignatures; i++) {
+            address currentAddress = accounts[i];
+            bytes memory signature = getSignature(
+                forSupport,
+                proposalId,
+                currentAddress,
+                keys[currentAddress]
+            );
+
+            if (isBaseline) {
+                baselineSignatures[currentAddress] = signature;
+            } else {
+                riscZeroSignatures[currentAddress] = signature;
+            }
+        }
+    }
+
+    function getJournal(
+        uint256 proposalId,
+        uint256 noOfAccounts
+    ) internal noGasMetering returns (bytes memory journal) {
+        (
+            bytes32 finalBallotBoxAccum,
+            bytes memory encodedBallots
+        ) = hashBallots(noOfAccounts);
+
+        journal = abi.encodePacked(
+            proposalId,
+            finalBallotBoxAccum,
+            encodedBallots
+        );
+    }
+
+    function getJournalDigest(
+        uint256 proposalId,
+        uint256 noOfAccounts
+    ) internal noGasMetering returns (bytes32 journalDigest) {
+        bytes memory journal = getJournal(proposalId, noOfAccounts);
+        journalDigest = sha256(journal);
+    }
+
+    function hashBallots(
+        uint256 noOfAccounts
+    ) internal noGasMetering returns (bytes32, bytes memory) {
+        bytes memory encodedBallots;
+        bytes32 ballotHash = 0x296dc540e823507aa12a2e7be3c9c01672a7d9bb7840214223e8758fdb2986c7;
```

Comment:
Where does this value come from? Could you add a comment or generate it automatically?

Reply:
alright, I took a look and it was actually very easy to find that there is a salt

```solidity
        // Write the proposal id to the ballot box commitment to act as a salt.
        _proposalVotes[id].ballotBoxCommit = bytes32(id);
```

I will generate this automatically now, thanks Victor!